### PR TITLE
Added new entries to AdvertFlags table

### LINKED ISSUE #487: Undocumented AdvertFlags
From closed pull request #409 by @davidpaMSFT 

> It seems we have some undocumented AdvertFlags. It seems these are for some deployment options added to the product later, e.g.:
>
>19 - "Allow task sequence to run for client on the Internet"
>28 - Deployment Settings\Make available for the following - "Only media and PXE"
>29 - Deployment Settings\Make available for the following - "Only media and PXE (hidden)"
>
>The above flags and corresponding deployment options are based on my observations in the lab and some DL discussions, but should be double-checked and also validate if there are any addtiional un-documented flags.
>
>In addition to documenting the bits we probably need an additional verbiage under the table about bits 28 and 29 - although setting these bits (without bit 18) will be reflected in UI as expected, the PXE (and likely media) clients will not see this advertisement/deployment unless bit 18 is also set.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3a4e1c6f-f511-82da-26ce-942618e89408
* Version Independent ID: 884eabf1-5a3f-7deb-a523-2d077b5ef648
* Content: [SMS_Advertisement Class - Configuration Manager](https://docs.microsoft.com/en-us/sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class)
* Content Source: [sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class.md)
* Service: **unspecified**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### PR DESCRIPTION
**Pull request by ChrisKibble**
Fixes #487

### CODE REVIEW

**Comment by PRMerger17** [comment]
@ChrisKibble : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class.md [hunk 1]
```diff
@@ -97,16 +97,20 @@ Class SMS_Advertisement : SMS_BaseClass
 |0x00000100 (8)|ONSYSTEMSTARTUP. Announce the advertisement to the user on system startup.|  
 |0x00000200 (9)|ONUSERLOGON. Announce the advertisement to the user on logon.|  
 |0x00000400 (10)|ONUSERLOGOFF. Announce the advertisement to the user on logoff.|  
+|0x00001000 (12)|OPTIONALPREDOWNLOAD. If the selected architecture and language matches that of the client, the package content will be downloaded in advance
 |0x00008000 (15)|WINDOWS_CE. The advertisement is for a device client.|  
-|0x00010000 (16)|ENABLE_PEER_CACHING <br /><br /> This information applies to System Center 2012 Configuration Manager SP1 or later, and System Center 2012 R2 Configuration Manager or later.|  
+|0x00010000 (16)|ENABLE_PEER_CACHING. This information applies to System Center 2012 Configuration Manager SP1 or later, and System Center 2012 R2 Configurcation Manager or later.|
```
Comment (aczechowski) [inline sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class.md]: oops, @ChrisKibble you have an extra "c" inserted here on Configuration

**Comment by PRMerger10** [comment]
@ChrisKibble : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ChrisKibble** [comment]
Guess I need Spell Check for Markdown in Code next :-)

File: sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class.md (hunk 1 above)
Comment (ChrisKibble) [reply sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class.md]: Resolved in new commit.

**Comment by jborsecnik** [comment]
@aczechowski, is this ok to merge now?

**Pull request merged**